### DISCUSSION TITLE
Lock nuget.config from overrides in subfolders

### DISCUSSION THREAD

**Discussion by sebastianslutzky** [NuGet/Home#11080] [Q&A] [answered]
Hi, 
I'm trying to  set nuget sources on a CI server, from a generated nuget.config at the root folder.
I want to make sure no other source is added in the build so I search for the existence of any nuget.config recursively, and fail the build if there is one...it has to be removed first

Is there any other way of doing this? I want to make sure that additional sources are not added in nuget.config files placed in subfolders

**Comment by nkolev92** [comment, marked as answer]
The hierarchical configs work closest first, so there's really no way to prevent configs closer to the solution from affecting the operation.
